cli/mizu/configStructs/tapConfig.go

```diff
@@ -27,21 +26,21 @@ const (
 )
 
 type TapConfig struct {
-	AnalysisDestination    string   `yaml:"dest" default:"up9.app"`
-	SleepIntervalSec       int      `yaml:"upload-interval" default:"10"`
-	PodRegexStr            string   `yaml:"regex" default:".*"`
-	GuiPort                uint16   `yaml:"gui-port" default:"8899"`
-	Namespaces             []string `yaml:"namespaces"`
-	Analysis               bool     `yaml:"analysis" default:"false"`
-	AllNamespaces          bool     `yaml:"all-namespaces" default:"false"`
-	KubeConfigPath         string   `yaml:"kube-config"`
-	PlainTextFilterRegexes []string `yaml:"regex-masking"`
-	HideHealthChecks       bool     `yaml:"hide-healthchecks" default:"false"`
-	DisableRedaction       bool     `yaml:"no-redact" default:"false"`
-	HumanMaxEntriesDBSize  string   `yaml:"max-entries-db-size" default:"200MB"`
-	Direction              string   `yaml:"direction" default:"in"`
-	DryRun                 bool     `yaml:"dry-run" default:"false"`
-	EnforcePolicyFile      string   `yaml:"test-rules"`
+	AnalysisDestination          string   `yaml:"dest" default:"up9.app"`
+	SleepIntervalSec             int      `yaml:"upload-interval" default:"10"`
+	PodRegexStr                  string   `yaml:"regex" default:".*"`
+	GuiPort                      uint16   `yaml:"gui-port" default:"8899"`
+	Namespaces                   []string `yaml:"namespaces"`
+	Analysis                     bool     `yaml:"analysis" default:"false"`
+	AllNamespaces                bool     `yaml:"all-namespaces" default:"false"`
+	KubeConfigPath               string   `yaml:"kube-config"`
+	PlainTextFilterRegexes       []string `yaml:"regex-masking"`
+	HealthChecksUserAgentHeaders []string `yaml:"ignored-user-agents" default:"[]"`
```

Comment:
few comments:
* lets enable use to set it by using the --set command
* lets call it user agent black list
* in readme we can say something: "Mizu can ignore HTTP traffic by matching specific User-Agent header in request. For instance if you not interested seeing health checks from 'kube-probe' and 'prometheus' you can run 'mizu tap --set user-agent-bl=kube-probe --set user-agent-bl=prometheus

Reply:
1. I don't want to enable it vi a --set (it doesn't make any sense) [it will create confusion because it can't be the same name of parameter and we do not support it [in --set it should be singular `ignored-user-agent` and in settings it is `ignored-user-agents` (because in YAML we see list and in --set you are putting only one every tim
2. black list is term of the past we are not using black list and ignored is better IMO.
3. is not relevant after what I say in 

I can do the changes (it is not big) but let me know what do you thing